server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/mail/EmailQuery.scala

```diff
@@ -37,16 +38,50 @@ object Limit {
 }
 case class QueryState(value: String) extends AnyVal
 
+sealed trait SortProperty {
+  def toSortClause: SortClause
+}
+case object ReceivedAtSortProperty extends SortProperty {
+  override def toSortClause: SortClause = SortClause.Arrival
+}
+
+object IsAscending {
+  val DESCENDING: IsAscending = IsAscending(false)
+}
+case class IsAscending(sortByASC: Boolean) extends AnyVal {
+  def toSortOrder: SearchQuery.Sort.Order = if (sortByASC) SearchQuery.Sort.Order.NATURAL else SearchQuery.Sort.Order.REVERSE
+
+}
+
+object Comparator {
+  val default: Comparator = Comparator(ReceivedAtSortProperty, Some(IsAscending.DESCENDING), None)
```

Comment:
No that's the contrary according to the spec, the default value for `isAcsending` should be true 

https://jmap.io/spec-core.html#query => sort section